src/snapred/backend/data/DataFactoryService.py

```diff
@@ -63,58 +49,92 @@ def getCalibrantSample(self, filePath):
     def getCifFilePath(self, sampleId):
         return self.lookupService.readCifFilePath(sampleId)
 
-    def getCalibrationState(self, runId, useLiteMode):
-        return self.lookupService.readCalibrationState(runId, useLiteMode)
+    def getSamplePaths(self):
+        return self.lookupService.readSamplePaths()
 
-    def getNormalizationState(self, runId):
-        return self.lookupService.readNormalizationState(runId)
+    def getGroupingMap(self, runId: str):
+        return self.lookupService.readGroupingMap(runId)
 
-    def writeNormalizationState(self, runId):
```

Comment:
This one I did delete.  It doesn't belong in the data factory service, but in the export service.